website/versioned_docs/version-0.80/turbo-native-modules-android.md

```diff
@@ -332,6 +338,12 @@ class MainApplication : Application(), ReactApplication {
 </TabItem>
 </Tabs>
 
+Before building and running your code on an emulator, we recommend cleaning your Android build to avoid potential issues:
+
+```bash
+cd android && ./gradlew clean
+```
+
```

Comment:
Let's remove this. you shouldn't need to `gradlew clean` anytime. If that is needed, it means there is an underlying problem.

Asking users to clean before building will just slow down builds unnecessarily